### ISSUE TITLE
Change from js to typescript.

### ISSUE THREAD

**Issue by vakshit** [TheTeamEngineer/ElectroTech#26]
Typescript is better. Because of this getting runtime errors is less likely, whereas JavaScript is an interpreted language. TypeScript supports static/strong typing. This means that type correctness can be checked at compile time. This feature is not available in JavaScript.

**Comment by Furquan712** [comment]
@vakshit would you like to work on this issue?

**Comment by vakshit** [comment]
Yup, i can work on it. But it would take time as I currently have a decent workload at college.

**Comment by Furquan712** [comment]
ok take  your tym

**Comment by Gurdeep475** [comment]
I would like to work on this issue under GSSOC`22. I have contributed on many projects having code base in TypeScript.
